hs_core/management/commands/fix_broken_resources.py

```diff
@@ -74,3 +77,46 @@ def handle(self, *args, **options):
             resource_modified(r, r.creator)
             # Do a redundant check to ensure that cleaning worked.
             r.check_irods_files(echo_errors=True, log_errors=False, return_errors=False)
+
+        print("REPAIR BROKEN NETCDF RESOURCES")
+
+        for r in BaseResource.objects.filter(resource_type='NetcdfResource'):
+            if r.is_federated and not settings.REMOTE_USE_IRODS:
+                print("INFO: skipping repair of federated resource {} in unfederated mode"
+                      .format(r.short_id))
+            else:
+
+                istorage = r.get_irods_storage()
+                for f in r.files.all():
+                    try:
+                        if f.short_path.endswith('.txt') and not istorage.exists(f.storage_path):
```

Comment:
@alvacouch Can we replace ```f.short_path``` above with ```f.storage_path``` instead? If so, it is probably a bit more efficient to assign ```f.storage_path``` to a variable, and use it inside this loop block rather than directly calling the property each time which would trigger some unnecessary overhead? The same applies to ```f.short_path``` as well since ```f.short_path``` appears to involve more checks. Also, could you remind me a bit on the difference between ```short_path``` and ```storage_path```? I can kind of see the difference by looking at the code, but want to get your input for confirmation especially on your thought behind the need when you added both properties.